modules/RTC/ortc/sdpUtils.js

```diff
@@ -0,0 +1,348 @@
+import sdpTransform from 'sdp-transform';
+
+/**
+ * Extract MID values from remote description.
+ * @param {Object} sdpObject - Remote SDP object generated by sdp-transform.
+ * @return {map<String, String>} Ordered Map with MID as key and kind as value.
+ */
+export function extractMids(sdpObject) {
+    const midToKind = new Map();
+
+    // Ignore disabled media sections.
+    for (const m of sdpObject.media) {
+        midToKind.set(m.mid, m.type);
+    }
+
+    return midToKind;
+}
+
+/**
+ * Extract ICE parameters from remote description.
+ * NOTE: This implementation assumes a single BUNDLEd transport.
+ * @param {Object} sdpObject - Remote SDP object generated by sdp-transform.
+ * @return {RTCIceParameters}
+ */
+export function extractIceParameters(sdpObject) {
+    let media;
+
+    // Ignore disabled media sections.
+    for (const m of sdpObject.media) {
```

Comment:
I meant to create a function to extract the first non-disabled media stream which has a given property. I count 3 occurrences.